internal/cli/quickstarts.go

```diff
@@ -355,10 +354,6 @@ func defaultURL(url string, port int) string {
 }
 
 func (i *qsInputs) fromArgs(cmd *cobra.Command, args []string, cli *cli) error {
-	if !canPrompt(cmd) {
-		return errors.New("this command can only be run on interactive mode")
-	}
-
```

Comment:
This conditional was originally added in #119 . It notes:

> **Returned an error when running in non-interactive mode**
> - The use case for this command only makes sense when run by a human being
> - This can be changed later if other use cases pop up

I consider the removal of this check appropriate to enable testing IMO.